ome/omero-figure
# License expression

Use valid SPDX license expression

To test: 
 * run ``python setup.py sdist``
 * Check that there is no message like ``npm WARN omero-figure@2.0.1 license should be a valid SPDX license expression
`` 
 see  https://spdx.org/licenses/
